airbyte-scheduler/client/src/main/java/io/airbyte/scheduler/client/SchedulerJobClient.java

```diff
@@ -24,7 +25,9 @@ Job createOrGetActiveSyncJob(SourceConnection source,
                                StandardSync standardSync,
                                String sourceDockerImage,
                                String destinationDockerImage,
-                               List<StandardSyncOperation> standardSyncOperations)
+                               List<StandardSyncOperation> standardSyncOperations,
+                               ActorDefinitionResourceRequirements sourceResourceRequirements,
```

Comment:
add java doc that these can be null.

Reply:
Could use the `@Nullable` annotation too

Reply:
Added `@Nullable` to these